packages/firestore/test/integration/util/helpers.ts

```diff
@@ -276,3 +277,17 @@ export function withTestCollectionSettings(
     }
   );
 }
+
+// TODO(ne-queries): This exists just so we don't have to do the cast
+// repeatedly. Once we expose '!=' publicly we can remove it and
+// just use 'array-contains-any' in all the tests.
```

Comment:
Typo? Surely you don't mean "array-contains-any" here (or "in" in the next one).